### DISCUSSION TITLE
Any repo banner ideas?

### DISCUSSION THREAD

**Discussion by BashhScriptKid** [Hafizat/PLW#5] [Ideas]
I actually don't really care about the banner but I think it looks better so, @sanikdah you got any ideas? Usually I would either create one myself or find one on the Internet but I was too lazy and have no idea for it

**Comment by techflashYT** [comment]
Banner?  Never really heard of banners on GitHub,  Can you explain?

**Comment by BashhScriptKid** [comment]
well something like this 
![Uploading Screenshot_20211219-074942.png…]()
